pkg/context/command_context.go

```diff
@@ -26,3 +26,19 @@ func NewCommandContext(term ioutils.Terminal, newClient NewClientFunc) *CommandC
 		NewClient: newClient,
 	}
 }
+
+// TerminalContext the context terminal utilities and KubeClient
+type TerminalContext struct {
+	context.Context
+	ioutils.Terminal
+	KubeClient runtimeclient.Client
+}
+
+// NewTerminalContext returns the context with the terminal utilities and the kubeClient to be used by the CLI command.
+func NewTerminalContext(term ioutils.Terminal, client runtimeclient.Client) *TerminalContext {
+	return &TerminalContext{
+		Context:    context.Background(),
+		Terminal:   term,
+		KubeClient: client,
```

Comment:
the fact that the context "carries" (or "contains") the client is a bit weird and looks unusual. 
I would rather extract it.

Reply:
yeah, I copied from the other contexts, but I've decoupled the client in https://github.com/kubesaw/ksctl/pull/48/commits/eba0d843d56cae9c555d7b3c8dac204d71dbcc85